src/marqo/s2_inference/model_registry.py

```diff
@@ -66,6 +66,190 @@ def _get_clip_properties() -> Dict:
 
         }
     return CLIP_MODEL_PROPERTIES
+def _get_open_clip_properties() -> Dict:
+    OPEN_CLIP_MODEL_PROPERTIES = {
+        'open_clip/RN50/openai':
+           {'name': 'open_clip/RN50/openai',
+            'dimensions': 512,
```

Comment:
the embedding dimension is not the same for all models - i don't have the reference for the actual values but worst case we could enumerate them all and get it